src/app/forms/templates/hgv/hgv-tech-record.template.ts

```diff
@@ -9,121 +15,150 @@ export const HgvTechRecord: FormNode = {
       name: 'vehicleType',
       label: 'Vehicle type',
       value: '',
-
+      width: FormNodeWidth.XS,
       type: FormNodeTypes.CONTROL,
-      viewType: FormNodeViewTypes.VEHICLETYPE
+      viewType: FormNodeViewTypes.VEHICLETYPE,
+      editType: FormNodeEditTypes.SELECT,
```

Comment:
I believe editing the vehicle type is out of scope as it is a lot more complex

Reply:
It is, both out of scope and a lot more complex, I just made it pretty, like you suggested I do in other places :o)